### ISSUE TITLE
chip icons

### ISSUE THREAD

**Issue by TheMaverickProgrammer** [chrisrsantiago/chiplibrary#13]
Would it be possible to add chip icons to the API? I'm writing an [open source net-battle engine](https://github.com/TheMaverickProgrammer/OpenNetBattle) that could replicate MMBN6. I release game demos every so often and since it's gaining popularity, I'm currently trying to avoid Capcom's IP infringement. Right now the engine downloads chip card images and icons. This way I don't ship the game with any copyrighted material and I can change the image by changing the URL on my web server that hosts the game content. Later it will also swap out the rest of the in-game battle art.

[see video](https://streamable.com/9vo0qm). notice the chip icon is an error icon 'X'. That's because my game doesn't point to any single icons (I can't find individual icons, only large sheets online).

I think your chip library is perfect for my use case, I could point all images to your API images. However I need icon images too (the ones you see in the chip selection screen). Is this something you can add?

**Comment by chrisrsantiago** [comment]
Agreed.  Not sure why this was not already present.  I. can go ahead and add them in the next commit, but can you refresh my memory on the following:

Are there any non navi chips that reuse icons?

**Comment by TheMaverickProgrammer** [comment]
No. Every non-navi chip has its own unique icon. Chips of the same name but different tiers, like Reflct1 vs Reflct 2 vs Reflct 3, will all have minor differences to them.

**Comment by chrisrsantiago** [comment]
Gotcha.  Thanks!

**Comment by TheMaverickProgrammer** [comment]
Any update on this? I'm going public today but not quite ready to distribute downloads yet.
